Add secret support to podman login (#19200)
containers/podman: docs/source/markdown/podman-secret-create.1.md

Secrets are not committed to an image with `podman commit`, and does not get committed in the archive created by a `podman export` command.

> You can also use secrets to store secret data whcih can be used by `podman login` to authenticate to container registries.

Reviewer: ```suggestion
You can also use secrets to store passwords for `podman login` to authenticate against container registries.
```
